docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
   - Click **Continue**.
 - Click the **Save** button and wait for Strapi to restart.
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
+
+- Click the **"+ Add Content Type"** button.
+- Enter `category`.
+- Click **"+ Add New Field"**
+  - Click the **String** field.
+  - Type `category` under the **BASE SETTINGS** tab, in the **Name** field.
+  - Check `Required field` and `Unique field` under the **ADVANCED SETTINGS** tab.
+  - Click **Continue**.
+- Click **"+ Add New Field"**
+  - Click the **Relation Field** field.
+  - On the right side, click the **Permissions** dropdown and select, `Restaurant`.
+  - In the center, select the icon the represents `many-to-many`. The text should read, `Categories has and belongs to Many Restaurants`.
+  - Click **Save**.
+- Click the **Save** button and wait for Strapi to restart.
+
+## 6. Add content to "Restaurant" Content Type
 
 Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
 
-- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Name** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
+- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Restaurant** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
 - Click **Save**.
 
 You will see your restaurant listed in the entries.
 
-## 6. Set roles and permissions
+## 7. Add categories to the "Category" Content Type
+
+Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/category?source=content-manager).
+
+- Click on **+ Add New Category** button. Type `Convenient` in the **Category** field. Select `Strapi Restaurant`, on the right, from **Restaurant (0)**
+- Click **Save**.
+
+You will see the **Convenient** category listed in the entries.
+
+- Click on **+ Add New Category** button. Type `Time Saving` in the **Category** field. **DO NOT ADD IT HERE** to `Strapi Restaurant`.
```

Comment:
i am not sure the **Do NOT ...** part is needed

Reply:
I know what you mean.  I added it because I felt it was super important to show that we develop relations from either content type.  Something, WordPress (as an example) doesn't really allow.  If they do add the category here, I am afraid they will get confused later.